src/mame/igs/pgm.cpp

```diff
@@ -5396,6 +5396,149 @@ ROM_START( ddpdojblkbl )
+| |   |                                                                               |
+| |   |              |------|   |------|   |------|   |------|   |------|   |------|  |
+| |   |              |      |   |      |   |      |   |      |   |      |   |      |  |
+| |---|              | COMM |   | MP3  |   | MP3  |   | MP3  |   | MP3  |   | FONT |  |
+|                    | U21  |   | 0-0  |   | 0-1  |   | 1-0  |   | 1-1  |   | CG   |  |
+|                    |      |   | U22  |   | U23  |   | U24  |   | U25  |   | U26  |  |
+|                    |------|   |------|   |------|   |------|   |------|   |------|  |
+|                                                                                     |
+|          |------|  |------|   |------|   |------|   |------|   |------|   |------|  |
+|          |      |  |      |   |      |   |      |   |      |   |      |   |      |  |
+|          | BANK |  | BANK |   | BANK |   | BANK |   | CG-3 |   | CG-4 |   | CG-5 |  |
+|          | 1-0  |  | 1-1  |   | 1-2  |   | 1-3  |   | U15  |   | U16  |   | U17  |  |
+|          | U11  |  | U12  |   | U13  |   | U14  |   |      |   |      |   |      |  |
+|          |------|  |------|   |------|   |------|   |------|   |------|   |------|  |
+|                                                                                     |
+|          |------|  |------|   |------|   |------|   |------|   |------|   |------|  |
+|          |      |  |      |   |      |   |      |   |      |   |      |   |      |  |
+|          | BANK |  | BANK |   | BANK |   | BANK |   | CG-0 |   | CG-1 |   | CG-2 |  |
+|          | 0-0  |  | 0-1  |   | 0-2  |   | 0-3  |   | U6   |   | U7   |   | U8   |  |
+|          | U2   |  | U3   |   | U4   |   | U5   |   |      |   |      |   |      |  |
+|          |------|  |------|   |------|   |------|   |------|   |------|   |------|  |
+|                                                                                     |
+|                                                                                     |
+|-------------------------------------------------------------------------------------|
+Notes:
+	  I am not sure what the size of these ROM chips are.
+	  They appear to be similar to other PGM-based games.
+*/
+ROM_START( rf1 )
+	ROM_REGION( 0x600000, "maincpu", 0 ) /* 68000 Code */
+	ROM_LOAD16_BYTE ( "rf1_sys_h.u32", 0x00000, 0x80000, CRC(e5e1e5f9) SHA1(1e8e4b7e53fcab68d14f36f9c8d85047f308bc93) )
+	ROM_LOAD16_BYTE ( "rf1_sys_l.u31", 0x00001, 0x80000, CRC(ee2c3817) SHA1(490e879f347ac38feaad80666d28fac9e2154188) )
+
+	ROM_REGION( 0x4000, "prot", ROMREGION_ERASE00 ) /* ARM protection ASIC - internal rom */
+	// ROM_LOAD( "rf1_igs027a.bin", 0x000000, 0x04000, NO_DUMP )
+
+	ROM_REGION( 0x800000, "tiles", ROMREGION_ERASE00 ) /* 8x8 Text Tiles + 32x32 BG Tiles */
+	// ROM_LOAD( "rf1_t2801.u26",     0x0000000, 0x0800000, NO_DUMP )
+
+	ROM_REGION16_LE( 0x2000000, "sprcol", ROMREGION_ERASE00 ) /* Sprite Colour Data */
```

Comment:
ROM_REGION sizes are smaller then the sum of the sizes of the ROMs you put in them. I.e. in this case the sprcol is 0x2000000 but you put 6x 0x800000 sized ROMs, which give 0x3000000 size for the region.

Reply:
I don't yet know the size of each ROM though, what should I be putting in instead?

Reply:
it's ok, you noted that above. Just leave them as 0x800000 each for now.